### DISCUSSION TITLE
Tested and approved!

### DISCUSSION THREAD

**Discussion by ikroeber** [vrtmrz/obsidian-livesync#106] [Show and tell]
Two days ago, I installed CouchDB on an AWS EC2 server of mine and tested obsidian-livesync. I've been using on both my desktop and on my android smartphone to sync my Obsidian notes and besides some issues regarding plugin sync, the notes were always magically synchronized.

I'm writing this to congratulate and share my satisfaction with the project. I will contribute as soon as I have time!!!

**Comment by vrtmrz** [comment]
Thank you for your post!
I am very glad to hear that! I am looking forward to hearing/seeing your contribution!

**Comment by c0deZ3R0** [comment]
I've just set up couchdb on a Rpi4 home server (running locally) with a wireguard tunnel! Early days, but it seems to be working flawlessly, love your work!
